packages/govuk-frontend/babel.config.js

```diff
@@ -11,6 +11,25 @@ module.exports = function (api) {
   const browserslistEnv = isBrowser ? 'javascripts' : 'node'
 
   return {
+    generatorOpts: {
+      shouldPrintComment (comment) {
+        if (!isBrowser || comment.includes('* @preserve')) {
+          return true
+        }
+
+        // Assume all comments are public unless
+        // tagged with `@private` or `@internal`
+        const isPrivate = ['* @internal', '* @private']
+          .some((tag) => comment.includes(tag))
+
+        // Flag any JSDoc comments worth keeping
+        const isDocumentation = ['* @param', '* @returns', '* @typedef']
```

Comment:
**question**: Would matching `/**` help us not have to add `@preserve` tags and limit the manual  flagging of comments we want to keep (like all the classes JSDoc blocks)?

Reply:
Sometimes we just have non-documentation comments like:

```mjs
/**
 * Hello Romaric
 */
```

So they get removed too